astroquery/xmatch/core.py

```diff
@@ -74,7 +74,8 @@ def query(self, cat1, cat2, max_distance,
                                     **kwargs)
         if get_query_payload:
             return response
-        return self._parse_text(response.text)
+
+        return Table.read(BytesIO(response.content), format='votable', use_names_over_ids=True)
```

Comment:
Minor suggestion, to make future debugging easier in case for some reason a failure happens in the content conversion step:
```suggestion
        content = BytesIO(response.content)
        return Table.read(content, format='votable', use_names_over_ids=True)
```
(though feel free to overrule this)